modules/cluster-autoscaler-about.adoc

```diff
@@ -15,7 +15,11 @@ The cluster autoscaler increases the size of the cluster when there are pods tha
 Ensure that the `maxNodesTotal` value in the `ClusterAutoscaler` resource definition that you create is large enough to account for the total possible number of machines in your cluster. This value must encompass the number of control plane machines and the possible number of compute machines that you might scale to.
 ====
 
-The cluster autoscaler decreases the size of the cluster when some nodes are consistently not needed for a significant period, such as when it has low resource use and all of its important pods can fit on other nodes.
+Every 10 seconds, the cluster autoscaler checks which nodes are unnecessary in the cluster and removes them. The cluster autoscaler considers a node for removal if the following conditions apply:
+
+* The sum of cpu and memory requests of all pods running on the node is smaller than 50% of the allocatable of a node.
+* The cluster autoscaler can move all pods running on the node to the other nodes.
+* The cluster autoscaler does not have scale-down disabled annotation.
```

Comment:
```suggestion
* The cluster autoscaler does not have scale-down disabled annotation.
```

Please verify whether "scale-down" should be hyphenated. I would double check with CLI/UI to make sure.

Reply:
Changing `scale-down` to `scale down` since elsewhere in the docs, it is mentioned as `scale down`.

Reply:
Acked